cfpb/cfgov-refresh
# Adding y axis label field to ChartBlock

Adding y axis label field to ChartBlock 



## Changes

- Modified code to add the y_axis_label field to the chart block. This provides content editors the ability to override the default label. 

## Testing

1. Update the `y-axis-label` field via Wagtail.
2. Preview the page and view source. 
3. Verify that the chart block data value has the value entered.

## Screenshots


## Notes

- This change corresponds to a change being made in the chart builder repo.

## Todos

-

## Checklist
